src/backend/tests/integration/astra/test_astra_component.py

```diff
@@ -186,6 +190,8 @@ def test_astra_vectorize_with_provider_api_key():
 )
 def test_astra_vectorize_passes_authentication():
     """tests vectorize using the authentication parameter"""
+    from langchain_astradb import AstraDBVectorStore, CollectionVectorServiceOptions
```

Comment:
ruff got it - need to add `from langflow.components.embeddings.AstraVectorize import AstraVectorizeComponent`